Bricks/mq/inmemory/test.cc

```diff
@@ -86,26 +91,31 @@ TEST(InMemoryMQ, DropOnOverflowTest) {
   for (size_t i = 0; i < 20; ++i) {
     mmq.PushMessage(bricks::strings::Printf("M%02d", static_cast<int>(i)));
   }
-  // Wait until at least three messages are processed by the consumer to
-  // properly get dropped messages count.
-  while (c.processed_messages_ < 2) {
-    ;  // Spin lock;
-  }
 
-  // Confirm that exactly 10 messages were dropped due to a very slow consumer.
+  // Confirm that exactly 10 messages were processed and exactly 10 were dropped due to a very slow consumer.
```

Comment:
This comment was related to a removed line: `EXPECT_EQ(c.dropped_messages_, 10u);`